Dockerfile.upgrade

```diff
@@ -31,7 +31,7 @@ RUN --mount=type=cache,target=/gocache \
     --mount=type=cache,target=/root/.cache/go-build \
     make build-local-edits
 
-RUN cp build/allorad /go/bin/allorad-v0.4.0
+RUN cp build/allorad /go/bin/allorad-v0.4.1
```

Comment:
needs to be v0.5.0

Reply:
https://github.com/allora-network/allora-chain/pull/594